### DISCUSSION TITLE
Can this be used on the axis-q6215-le? ARTPEC-6 chipset?

### DISCUSSION THREAD

**Discussion by whittenator** [AxisCommunications/acap-computer-vision-sdk-examples#99] [Q&A] [answered]
Can the Object Detection be ran on the ARTPEC-6 chipset of the Q6215-LE?

**Comment by Corallo** [comment, marked as answer]
Hi @whittenator 
Artpec-6 is not supported by the computer vision SDK examples and also not supported by our inference lib, so unfortunately this is not possible.

We recommend running these kinds of applications on artpec 7 or 8 cameras that are equipped with hardware accelerators.
